content/docs/concepts.md

```diff
@@ -0,0 +1 @@
+See https://kudo.dev/docs/what-is-kudo.html#under-the-hood

```

Comment:
Can you make this a redirect in [netlify.toml](https://github.com/kudobuilder/kudo.dev/blob/master/netlify.toml) instead?

Reply:
Done, PTAL.